frontend/test/resolution/testArrays.cpp

```diff
@@ -43,8 +43,11 @@ static QualifiedType findVarType(const Module* m,
 
 static void testArray(std::string domainType,
                       std::string eltType) {
-  Context ctx;
+  Context::Configuration config;
+  config.chplHome = getenv("CHPL_HOME");
+  Context ctx(config);
   Context* context = &ctx;
+  setupModuleSearchPaths(context, false, false, {}, {});
```

Comment:
Hm, this test starts failing when I use `buildStdContext`, apparently due to it running `publicSymbolsForModule` on `ChapelStandard` causing the real `ChapelArray` to be used instead of the minimal version. It's a little unfortunate but since I'm not trying to resolve ChapelArray in this PR I might have to leave it as is. I tried advancing the revision or using `setPrependedInternalModulePath` to get around it but no dice.